README.md

```diff
@@ -111,22 +115,18 @@ The pipeline is configured by a JSON5 structure with top-level elements
 
 ````text
 {
-    type {String} The name of the overall pipeline (will eventually become a grade name of some sort) 
+    type {String} The name of the overall pipeline - a grade name
+    parents {String|String[]} Any parent pipelines that this pipeline should be composited together with
      
-    datasets: { // Free hash of dataset structures each with elements
-        sourceType: "git", (currently the only supported type)
-        repository: <git repository URL, e.g. "https://github.com/inclusive-design/covid-assessment-centres">
-        path: <String path within the repository, e.g. "WeCount/assessment_centre_data_collection_2020_09_02.csv">
-    },
-
-    pipeline: { // Free hash of pipeline elements with elements
+    elements: { // Free hash of pipeline elements
```

Comment:
The `elements` block contains keyed elements. Properties at line 122-124 are supported at the element level instead of at the `elements` level.